src/shared/ui/list/list.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react';
+import { List, cx } from './list';
+
+const data = ['listItem 1', 'listItem 2', 'listItem 3'];
+
+describe('List Component', () => {
+  it('renders list correctly', () => {
+    render(<List data={data} />);
+
+    const list = screen.getByTestId('list');
+
+    expect(list).toBeVisible();
+  });
```

Comment:
Can't it be combined with `renders list-items correctly` and `renders all list items`?